etk-analyze/Cargo.toml

```diff
@@ -1,18 +1,24 @@
 [package]
 name = "etk-analyze"
 version = "0.1.0"
-authors = ["Sam Wilson <sam.wilson@mesh.xyz>"]
+authors = ["Sam Wilson <sam.wilson@mesh.xyz>", "lightclient <lightclient@protonmail.com>"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
+description = "Ethereum Toolkit analysis tools"
```

Comment:
I think "tool kit" is actually correct: https://www.copyeditcat.com/2014/05/08/editors-tool-kit/#:~:text=Sure%2C%20toolkit%20looks%20correct%2C%20but%20maybe%20it's%20really%20two%20words.&text=According%20to%20the%20Canadian%20Oxford,indeed%20two%20words%3A%20tool%20kit.